addon/helpers/string.js

```diff
@@ -3,6 +3,16 @@ import { MicroState } from 'ember-microstates';
 
 export default MicroState.extend({
 
+  initialValueFor([string]) {
+    if (string === undefined || string === null) {
```

Comment:
We can achieve the final two arms of this `if` with a simple `String(string)` which will properly coerce it.

And then we can achieve the first arm of the `if` by having a default value for string: `[string = '']` in the function parameters.

So I think we can bring this back to a one-liner.